README.rst

```diff
@@ -8,49 +8,34 @@ environments to encapsulate the build process.
 You can find these images on Docker Hub, on the `readthedocs/build`_
 repository:
 
-`readthedocs/build:1.0`
-    **Deprecated**
-    Ubuntu 14.04 based image.
+`readthedocs/build:ubuntu-20.04-YYYY.MM.DD`
+    Ubuntu 20.04 supporting multiple versions of Python, PyPy, conda, mamba, nodejs, rust and go.
+    Available for public usage as ``build.os: ubuntu-20.04``
 
-`readthedocs/build:1.0-dotnet`
-    **Deprecated**
-    Ubuntu 14.04 based image, plus .NET.
+Note that these images only contains the basic dependencies:
 
-`readthedocs/build:2.0`
-    **Deprecated**
-    Ubuntu 16.04 based image.
-
-`readthedocs/build:4.0`
-    **Deprecated**
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7.
-
-`readthedocs/build:5.0`
-    ``stable``
-    Ubuntu 18.04 supporting Python 2.7, 3.6, 3.7 and pypy3.5-7.0.0.
-    This is the **stable** image supported by Read the Docs.
-
-`readthedocs/build:6.0`
-    ``latest``
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
-    This is the **latest** default image used for documentation builds and supported by Read the Docs.
-
-`readthedocs/build:7.0`
-    ``testing``
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9, and PyPy3.5-7.0.0.
-    Available for public usage as **testing** image. You should expect some breaking changes here.
```

Comment:
Are these not still important to document, since they are still in use?

Reply:
This was a little (too much) opinionated 😄 

- I think we shouldn't explain these images here "to the final user"
- I think users wanting to _use_ these images should be pointed to the Read the Docs documentation instead (config file)
- I would try to keep this repository hidden from the regular users because it only generates confusion
- I wanted to only expose the newer image to avoid people finding this and using the old images

That said, it may make sense to remove the phrase "Available for public usage as ``build.os: ubuntu-20.04``" as well.

Reply:
> I would try to keep this repository hidden from the regular users because it only generates confusion

Perhaps we should make it private, then? And move these docs to our public-facing docs & config file?

Reply:
Make it private may be too much. I think we will lose some issues/prs references linked everywhere.

Definitely, how to use the docker image and what it contains should be documented in the Read the Docs documentation for the config file.